lib/srv/db/postgres/test.go

```diff
@@ -145,8 +145,20 @@ func (s *TestServer) handleConnection(conn net.Conn) error {
 		s.log.Debugf("Received %#v.", message)
 		switch msg := message.(type) {
```

Comment:
```suggestion
		switch message.(type) {
```

Reply:
`msg` is actually being used by the default clause below (to log the message).

Reply:
You can use `message` in that log instead